config/main.go

```diff
@@ -24,71 +24,73 @@ type EntitlementsConfig struct {
 
 // EntitlementsConfigKeysType is the definition of the struct hat houses all the env variables key names
 type EntitlementsConfigKeysType struct {
-	Key             string
-	Cert            string
-	Port            string
-	LogLevel        string
-	CertsFromEnv    string
-	SubsHost        string
-	ComplianceHost  string
-	CaPath          string
-	OpenAPISpecPath string
-	BundleInfoYaml  string
-	CwLogGroup      string
-	CwLogStream     string
-	CwRegion        string
-	CwKey           string
-	CwSecret        string
-	Features        string
-	FeaturesPath    string
-	SubAPIBasePath  string
-	CompAPIBasePath string
-	RunBundleSync   string
-	EntitleAll      string
-	AMSHost         string
-	ClientID        string
-	ClientSecret    string
-	TokenURL        string
-	Debug           string
-	BOPClientID     string
-	BOPToken        string
-	BOPURL          string
-	BOPEnv          string
-	BOPMockOrgId    string
+	Key                string
+	Cert               string
+	Port               string
+	LogLevel           string
+	CertsFromEnv       string
+	SubsHost           string
+	ComplianceHost     string
+	CaPath             string
+	OpenAPISpecPath    string
+	BundleInfoYaml     string
+	CwLogGroup         string
+	CwLogStream        string
+	CwRegion           string
+	CwKey              string
+	CwSecret           string
+	Features           string
+	FeaturesPath       string
+	SubAPIBasePath     string
+	CompAPIBasePath    string
+	RunBundleSync      string
+	EntitleAll         string
+	AMSHost            string
+	ClientID           string
+	ClientSecret       string
+	TokenURL           string
+	Debug              string
+	BOPClientID        string
+	BOPToken           string
+	BOPURL             string
+	BOPEnv             string
+	BOPMockOrgId       string
+	DisableSeatManager string
```

Comment:
all these are whitespace changes from "string" being indented outward. the only real change is the new type at the bottom. same with the keys  below